scapy/contrib/automotive/enumerator.py

```diff
@@ -538,6 +538,7 @@ def __get_retry_iterator(self):
     def __get_initial_request_iterator(self, state, **kwargs):
         # type: (EcuState, Any) -> Iterable[Packet]
         if state not in self._request_iterators:
+            kwargs["state"] = state
```

Comment:
Could you pleas verify, if "state" already exists? And maybe raise an exception in case you would overwrite it... Otherwise someone might search a long time for a potential bug.

Reply:
I don't see this resolved.

Reply:
Ups, sorry. Forgotten to remove it, even though it was not necessary anymore. Changed